proof/crefine/Move_C.thy

```diff
@@ -325,19 +151,11 @@ lemma unat_ucast_prio_shiftr_simp[simp]:
   "unat (ucast (p::priority) >> n :: machine_word) = unat (p >> n)"
   by simp
 
-lemma from_bool_to_bool_and_1 [simp]:
-  assumes r_size: "1 < size r"
-  shows "from_bool (to_bool (r && 1)) = r && 1"
```

Comment:
Was this a duplicate / had already been moved at some point?

Reply:
It's sort-of a duplicate in the sense that the assumption is not needed and the conclusion is an instance of the `from_bool_to_bool_and_1` that this one would shadow.